Content.Shared/Storage/EntitySystems/SharedStorageSystem.cs

```diff
@@ -1384,7 +1384,12 @@ public ItemSizePrototype GetMaxItemSize(Entity<StorageComponent?> uid)
 
         // If we specify a max item size, use that
         if (uid.Comp.MaxItemSize != null)
-            return _prototype.Index(uid.Comp.MaxItemSize.Value);
+        {
+            if (_prototype.TryIndex(uid.Comp.MaxItemSize.Value, out var proto))
```

Comment:
MaxItemSize is already ProtoId so yaml isnt allowed to make it invalid

the actual cause is somewhere in storage system that changes it to some random string